tests/foreman/destructive/test_contentview.py

```diff
@@ -19,62 +22,68 @@
 pytestmark = pytest.mark.destructive
 
 
+@pytest.fixture(scope='module')
+def module_big_repos(module_target_sat, module_sca_manifest_org):
+    """Enables and syncs three bigger RH repos"""
+    repos = []
+    for tag in ['rhel7_optional', 'rhel7_extra', 'rhel7_sup']:
+        id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+            basearch=constants.DEFAULT_ARCHITECTURE,
+            org_id=module_sca_manifest_org.id,
+            product=constants.REPOS[tag]['product'],
+            repo=constants.REPOS[tag]['name'],
+            reposet=constants.REPOS[tag]['reposet'],
+            releasever=constants.REPOS[tag]['releasever'],
+        )
+        repo = module_target_sat.api.Repository(id=id).read()
+        repos.append(repo)
+        repo.sync(synchronous=False)
+    module_target_sat.wait_for_tasks(
+        search_query=(
+            f'label = Actions::Katello::Repository::Sync and organization_id = {module_sca_manifest_org.id}'
+        ),
+        poll_timeout=750,
+        search_rate=10,
+        max_tries=75,
+    )
+    return repos
+
+
 @pytest.mark.tier4
 @pytest.mark.run_in_one_thread
-def test_positive_reboot_recover_cv_publish(target_sat, function_entitlement_manifest_org):
+@pytest.mark.parametrize('reboot', [True, False], ids=['vm_reboot', 'fm_restart'])
+def test_positive_reboot_recover_cv_publish(
+    target_sat, module_sca_manifest_org, module_big_repos, reboot
```

Comment:
```suggestion
    module_target_sat, module_sca_manifest_org, module_big_repos, reboot
```
`module_big_repos` is using `module_target_sat` and the test is using `target_sat`, seems like a mistake.
 Same suggestion for the rest of the `target_sat` occurrences.

Reply:
Yeah, but I guess it won't help since we are a bit [destructive here](https://github.com/SatelliteQE/robottelo/pull/14782/files#diff-1ffbb7e36ad174ce34c6c1e5b7cae69454ba5d5ace92c05020f8f5d6b19c86b6R22), so we get new Satellite for each test run [here](https://github.com/SatelliteQE/robottelo/blob/db19610b89dd4577b59f27555382c4aa736eaa3d/pytest_fixtures/core/broker.py#L25), which means we need new setup for each parametrized test which makes me cry and doubt if I really want to do the setup twice just to check the `f-m restart`.

Reply:
Okay so it seems it did help, PRT passed.